README.rst

```diff
@@ -4,109 +4,114 @@ NVIDIA DALI
 
 ----
 
-Examples and tutorials
+Examples and Tutorials
 ----------------------
 
-Introduction to DALI can be found at |dali_start|_ docs page.
+An introduction to DALI can be found in the |dali_start|_ page.
 
-The `docs/examples <https://github.com/NVIDIA/DALI/blob/master/docs/examples>`_
-directory contains a few examples (in the form of Jupyter notebooks)
-highlighting different features of DALI and how to use DALI to interface
-with deep learning frameworks.
+More advanced examples can be found in the |dali_examples|_ page.
 
-.. |dali_start| replace:: Getting started
-.. _dali_start: https://docs.nvidia.com/deeplearning/dali/user-guide/docs/examples/getting%20started.html
+For an interactive version (Jupyter notebook) of the examples, go to the `docs/examples <https://github.com/NVIDIA/DALI/blob/master/docs/examples>`_
+directory.
 
-Also note:
+**Note:** Select the |release-doc|_ or the |nightly-doc|_, which stays in sync with the main branch,
+depending on your version.
 
-- Documentation for the latest stable release is available
-  |here1|_, and
-- Nightly version of the documentation that stays in sync with the
-  main branch is available |here2|_.
-
-.. |here1| replace:: here
-.. _here1: https://docs.nvidia.com/deeplearning/dali/user-guide/docs/index.html
-.. |here2| replace:: here
-.. _here2: https://docs.nvidia.com/deeplearning/dali/master-user-guide/docs/index.html
+.. |dali_start| replace:: Getting Started
+.. _dali_start: https://docs.nvidia.com/deeplearning/dali/user-guide/docs/examples/getting%20started.html
+.. |dali_examples| replace:: Examples and Tutorials
+.. _dali_examples: https://docs.nvidia.com/deeplearning/dali/user-guide/docs/examples/index.html
+.. |release-doc| replace:: Latest Release Documentation
+.. _release-doc: https://docs.nvidia.com/deeplearning/dali/user-guide/docs/index.html
+.. |nightly-doc| replace:: Nightly Release Documentation
+.. _nightly-doc: https://docs.nvidia.com/deeplearning/dali/master-user-guide/docs/index.html
 
 ----
 
-Additional resources
+Additional Resources
 --------------------
 
 - GPU Technology Conference 2018; Fast data pipeline for deep learning training, T. Gale, S. Layton and P. Trędak: |slides1|_, |recording1|_.
 - GPU Technology Conference 2019; Fast AI data pre-preprocessing with DALI; Janusz Lisiecki, Michał Zientkiewicz: |slides2|_, |recording2|_.
 - GPU Technology Conference 2019; Integration of DALI with TensorRT on Xavier; Josh Park and Anurag Dixit: |slides3|_, |recording3|_.
 - GPU Technology Conference 2020; Fast Data Pre-Processing with NVIDIA Data Loading Library (DALI); Albert Wolant, Joaquin Anton Guirao |recording4|_.
 - `Developer page <https://developer.nvidia.com/DALI>`_.
-- `Blog post <https://devblogs.nvidia.com/fast-ai-data-preprocessing-with-nvidia-dali/>`_.
+- `Blog post: Fast AI Data Preprocessing with NVIDIA DALI <https://devblogs.nvidia.com/fast-ai-data-preprocessing-with-nvidia-dali/>`_.
```

Comment:
Maybe we can refer all blog posts related to DALI - https://developer.nvidia.com/blog/tag/dali/ ?

Reply:
Done